features/git-town/configuration/configuration_automatic.feature

```diff
@@ -9,10 +9,43 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     Given I haven't configured Git Town yet
 
 
-  Scenario Outline: Proceeding to configuration upon initial config prompt
+  Scenario Outline: Seeing a configuration prompt when running a Git Town command
```

Comment:
Scenario Outline: All Git Town commands show the configuration wizard if running unconfigured